src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -8715,6 +8715,20 @@ public static partial class File
         public static void WriteAllText(string path, string? contents, System.Text.Encoding encoding) { }
         public static System.Threading.Tasks.Task WriteAllTextAsync(string path, string? contents, System.Text.Encoding encoding, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public static System.Threading.Tasks.Task WriteAllTextAsync(string path, string? contents, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public static void SetCreationTime(Microsoft.Win32.SafeHandles.SafeFileHandle fileHandle, DateTime creationTime) { }
```

Comment:
I would expect these to be in alphabetical order. did you try this ? https://github.com/dotnet/runtime/blob/3535e0769f202ae4cd820bea24afd20cee313966/docs/coding-guidelines/updating-ref-source.md#for-systemruntime
I know sometimes it doesn't produce all the same output (due to bugs) but it should do enough to show you where they should go.

Reason for ordering is that ideally, when running the above command, it would not affect existing entries.

Reply:
This was generated by this generator :/

Reply:
uh, @ViktorHofer is this right?

But, if that's what it generates, it's fine for this PR.

Reply:
I think they may have come from a commit you cherry-picked from me, and I didn't use the generator.